__tests__/booking-algorithm/contiguous-slot-run.test.ts

```
@@ -0,0 +1,155 @@
+      startsAt,
+      durationInHours: 2,
+      consultantProfileId: "cp_1",
+      isTentative: false,
+    });
+    expect(atoms).toHaveLength(4);
+    for (let i = 0; i < atoms.length; i++) {
+      const expectedStart = startsAt.getTime() + i * SLOT_DURATION_MS;
+      expect(atoms[i].startsAt.getTime()).toBe(expectedStart);
+      expect(atoms[i].endsAt.getTime()).toBe(expectedStart + SLOT_DURATION_MS);
+      if (i > 0) {
+        expect(atoms[i].startsAt.getTime()).toBe(atoms[i - 1].endsAt.getTime());
```

ESLint

[error] Avoid calling `expect` conditionally`

(jest/no-conditional-expect)